src/index.ts

```diff
@@ -8,12 +8,17 @@ import { Exporter } from "./exporter"
 
 const program = new Command("askar-tools-javascript")
 
+export enum Strategy {
+  EXPORT = 'export',
+  DB_PW_EXPORT = 'db-pw-export'
+}
+
 program
   .requiredOption(
     "--strategy <strategy>",
     "Specify strategy to be used. Choose from 'mt-convert-to-mw', or 'import-tenant'.",
     (value) => {
-      if (!["mt-convert-to-mw", "import-tenant","export"].includes(value)) {
+      if (!["mt-convert-to-mw", "import-tenant","export","db-pw-export"].includes(value)) {
```

Comment:
I think should not use extra case as `db-pw-export` is a case under a `export` only